app/.yarnrc

```diff
@@ -0,0 +1,5 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
+yarn-offline-mirror "./vendor/cache"
```

Comment:
Should this be changed to `../vendor/cache` so any packages with the same version aren’t duplicated?

Reply:
Sadly no. I initially had this as just a root `.yarnrc` and what would happen was roughly this:

 - `yarn install` runs against `package.json`
 - when complete, unneeded packages _for `package.json`_ would be removed
 - `cd app && yarn install` runs against `app/package.json`
 - when complete, unneeded packages _for `app/package.json`_ would be removed

So you'd end up in a state where whoever ran last would have their packages in the cache, and the other will be missing some packages it needs (and `yarn install --offline` would error). Just seems to be a design limitation that we can't workaround currently.